File: EjercicioArreglo.cpp

#include <iostream>
using namespace std;

int main() {

    int numeros[4] = {10, 20, 30, 40};

    cout << "Primer número: " << numeros[0] << endl;

    for (int i = 0; i < 4; i++) {
        cout << numeros[i] << endl;
    }

    return 0;
}

#include <iostream>
#include <vector>
using namespace std;

int main() {

    vector<int> numeros;

    numeros.push_back(10);
    numeros.push_back(20);
    numeros.push_back(30);

    cout << "Primer número: " << numeros[0] << endl;

    for (int num : numeros) {
        cout << num << endl;
    }

    return 0;
}

File: EjercicioArreglo.java
Código Java:
public class Arreglos {
    public static void main(String[] args) {

        int[] numeros = {10, 20, 30, 40};

        System.out.println("Segundo número: " + numeros[1]);

        for (int i = 0; i < numeros.length; i++) {
            System.out.println(numeros[i]);
        }
    }
}public class Arreglos {
    public static void main(String[] args) {

        int[] numeros = {10, 20, 30, 40};

        System.out.println("Segundo número: " + numeros[1]);

        for (int i = 0; i < numeros.length; i++) {
            System.out.println(numeros[i]);
        }
    }
}

import java.util.ArrayList;
public class Arraylist {

	public static void main(String[] args) {
		 ArrayList<Integer> numeros = new ArrayList<>();

	        numeros.add(10);
	        numeros.add(20);
	        numeros.add(30);

	        System.out.println("Primer número: " + numeros.get(0));

	        for (int num : numeros) {
	            System.out.println(num);
	        }

	}

}
